pinc/ProjectTransition.inc

```diff
@@ -587,178 +942,257 @@ for ($rn = 1; $rn <= MAX_NUM_PAGE_EDITING_ROUNDS; $rn++) {
                 'detour' => null,
-                'settings_template' => "modifieddate='<TIMESTAMP>', checkedoutby=IF(checkedoutby!='',checkedoutby,username)",
-                'collateral_actions' => 'round_to_PP_out_collateral',
+                'settings_template' => "modifieddate='<TIMESTAMP>'",
+                'collateral_actions' => null,
             ]
         );
     }
-
-    if (1) {
-        // Transitions to skip rounds
-
-        $from_states = [
-            $round->project_waiting_state,
-            $round->project_unavailable_state,
-        ];
-
-        foreach ($from_states as $from_state) {
-            if ($rn < MAX_NUM_PAGE_EDITING_ROUNDS) {
-                $next_round = get_Round_for_round_number($rn + 1);
-
-                new ProjectTransition(
-                    $from_state,
-                    $next_round->project_waiting_state,
-                    'site_manager',
-                    [
-                        'project_restriction' => null,
-                        'action_name' => _('Skip this round'),
-                        'confirmation_question' => '[default]',
-                        'detour' => null,
-                        'settings_template' => "modifieddate='<TIMESTAMP>', nameofwork=CONCAT(nameofwork,' {{$round->id} skipped}')",
-                        'collateral_actions' => null,
-                    ]
-                );
-            } else {
-                new ProjectTransition(
-                    $from_state,
-                    PROJ_POST_FIRST_AVAILABLE,
-                    'site_manager',
-                    [
-                        'project_restriction' => 'proj_would_go_to_PP_avail',
-                        'action_name' => _('Skip this round'),
-                        'confirmation_question' => '[default]',
-                        'detour' => null,
-                        'settings_template' => "modifieddate='<TIMESTAMP>', nameofwork=CONCAT(nameofwork,' {{$round->id} skipped}'), n_available_pages=0, checkedoutby=''",
-                        'collateral_actions' => 'round_to_PP_avail_collateral',
-                    ]
-                );
-
-                new ProjectTransition(
-                    $from_state,
-                    PROJ_POST_FIRST_CHECKED_OUT,
-                    'site_manager',
-                    [
-                        'project_restriction' => 'proj_would_go_to_PP_out',
-                        'action_name' => _('Skip this round'),
-                        'confirmation_question' => '[default]',
-                        'detour' => null,
-                        'settings_template' => "modifieddate='<TIMESTAMP>', nameofwork=CONCAT(nameofwork,' {{$round->id} skipped}'), n_available_pages=0, checkedoutby=IF(checkedoutby!='',checkedoutby,username)",
-                        'collateral_actions' => 'round_to_PP_out_collateral',
-                    ]
-                );
-            }
-        }
-    }
+    return $project_transitions;
 }
 
```

Comment:
Can we please add an "`<hr>`" here and a comment that the functions below this are used in the creation of ProjectTransition objects for either `project_restriction` or `collateral_actions` callbacks? eg:
```
// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

// The functions below are used as callbacks for ProjectTransitions, either
// as the project_restriction or the collateral_actions.

```